spock-specs/src/test/groovy/org/spockframework/smoke/extension/TimeoutExtension.groovy

```diff
@@ -29,13 +29,12 @@ import static java.util.concurrent.TimeUnit.*
  * @author Peter Niederwieser
  */
 class TimeoutExtension extends EmbeddedSpecification {
-  @Shared Thread testFrameworkThread = Thread.currentThread()
 
   def setup() {
     runner.addClassMemberImport TimeUnit
   }
 
-  @Timeout(1)
+  @Timeout(10)
```

Comment:
Please, do not post similar comments for similar problems. Otherwice I will be forsed to copy all my messages multiple times.

We have slow Shippable integration. It does not work find in parallel mode. 250ms is usual delay for thread switching on Shippable boxes.